addons/civ_placement/fnc_createRoadblock.sqf

```diff
@@ -62,25 +58,22 @@ if (_num > 5) then {_num = 5};
 // Find all the roads
 _roads = _pos nearRoads (_radius + 20);
 
-private _result = [];
-
 // scan road positions, filter trails, filter runways and find those roads on outskirts
-{
-    if (_x distance _pos < (_radius - 10) || {!isOnRoad _x} || {str(_x) find "invisible" != -1}) then {
-        _roads set [_foreachIndex,objNull];
-    };
-} foreach _roads;
-_roads = _roads - [objNull];
+_roads = _roads select {_x distance _pos < (_radius - 10) || {!isOnRoad _x} || {(str _x) find "invisible" != -1}};
```

Comment:
If i'm reading this correct, it will return the exact opposite of what the previous code returned, since the `_roads set [_foreachIndex,objNull];` and `_roads = _roads - [objNull];` ?!

Tried the changes with a devbuild, and didn't get any roadblocks, only a bunch of `ALiVE Roadblock <Obj-Null> is not on a road [0,0,0]! Not created...`

Reply:
You are totally right...